recipes/sqlite3/all/test_package/empty_vfs.c

```diff
@@ -0,0 +1,90 @@
+#include <sqlite3.h>
+#include <stddef.h>
+
+int empty_xOpen(sqlite3_vfs *vfs, const char *zName, sqlite3_file *f, int flags, int *pOutFlags)
+{
+    // TODO: implement
+    return SQLITE_OK;
+}
+
+int empty_xDelete(sqlite3_vfs *vfs, const char *zName, int syncDir)
+{
+    // TODO: implement
+    return SQLITE_OK;
+}
+
+int empty_xAccess(sqlite3_vfs *vfs, const char *zName, int flags, int *pResOut)
+{
+    // TODO: implement
+    return SQLITE_OK;
+}
+
+int empty_xFullPathname(sqlite3_vfs *vfs, const char *zName, int nOut, char *zOut)
+{
+    // TODO: implement
+    return SQLITE_OK;
+}
+int empty_xRandomness(sqlite3_vfs *vfs, int nByte, char *zOut)
+{
+    // TODO: implement
```

Comment:
![image](https://user-images.githubusercontent.com/4138939/115876874-1f517f00-a447-11eb-8218-7557fd1da0af.png)

Reply:
As you can see [here](https://sqlite.org/c3ref/vfs.html), `xRandomness()` should fill `nBytes` in `zOut`.

I think, that the default written value will be a nice random too, not you?

Reply:
Your implementation is completely fine!
I was making an obligatory xkcd reference.